CONTRIBUTING.md

```diff
@@ -152,82 +152,90 @@ Finally, ![GitHub labels](https://img.shields.io/github/labels/nipreps/EddyMotio
+    For example, to update your master branch on your local cloned repository:
+    `Shell git fetch upstream git checkout master git merge upstream/master `
 
 1. **Create a [new branch][link_branches] to develop and maintain the proposed code changes.**<br />
-  For example:
-    ```Shell
-    git fetch upstream  # Always start with an updated upstream
-    git checkout -b fix/bug-1222 upstream/master
-    ```
-    Please consider using appropriate branch names as those listed below, and mind that some of them
-    are special (e.g., `doc/` and `docs/`):
-      * `fix/<some-identifier>`: for bugfixes
-      * `enh/<feature-name>`: for new features
-      * `doc/<some-identifier>`: for documentation improvements.
-        You should name all your documentation branches with the prefix `doc/` or `docs/`
-        as that will preempt triggering the full battery of continuous integration tests.
+   For example:
+
+   ```Shell
+   git fetch upstream  # Always start with an updated upstream
+   git checkout -b fix/bug-1222 upstream/master
+   ```
+
+   Please consider using appropriate branch names as those listed below, and mind that some of them
+   are special (e.g., `doc/` and `docs/`):
+   _ `fix/<some-identifier>`: for bugfixes
+   _ `enh/<feature-name>`: for new features \* `doc/<some-identifier>`: for documentation improvements.
+   You should name all your documentation branches with the prefix `doc/` or `docs/`
+   as that will preempt triggering the full battery of continuous integration tests.
 
 1. **Make the changes you've discussed, following the [EddyMotionCorrection coding style guide](#EddyMotionCorrection-coding-style-guide).**<br />
-  Try to keep the changes focused: it is generally easy to review changes that address one feature or bug at a time.
-  It can also be helpful to test your changes locally,
-  using a [EddyMotionCorrection development environment][link_devel].
-  Once you are satisfied with your local changes, [add/commit/push them][link_add_commit_push]
-  to the branch on your forked repository.
+   Try to keep the changes focused: it is generally easy to review changes that address one feature or bug at a time.
+   It can also be helpful to test your changes locally,
+   using a [EddyMotionCorrection development environment][link_devel].
+   Once you are satisfied with your local changes, [add/commit/push them][link_add_commit_push]
+   to the branch on your forked repository.
+
+1. **Make sure your changes follow the project's styling configurations**.  <br />
+  To allow for continous integration and collaboration with other members of the community, it's important to keep stable styling configurations for a project. To make sure you follow these standard, run the following commands:
+    ```Shell
+      pip install pre-commit
+      pre-commit install
+    ```
+    Once the pre-commit hooks are installed and configured, each commit you'll make will be followed by a few simple styling "tests".
```

Comment:
After submitting my review, I re-read your description and realized I had missed this addition. This is very much appreciated, however buried within so many unrelated changes I just did not see it.

Please, include this change. Ideally, this file should only show this new entry as a change.